kukkuu/exceptions.py

```diff
@@ -124,5 +124,9 @@ class TicketSystemPasswordAlreadyAssignedError(KukkuuGraphQLError):
     """
 
 
+class TicketSystemPasswordNothingToImportError(KukkuuGraphQLError):
+    """Empty list of passwords given"""
```

Comment:
Not a huge deal and does not really matter in practice that much, but in principle, I think these specific errors should be reserved mostly for situations where the client needs to be able to identify the error programmatically (yeah, not all Kukkuu's current errors follow that either 😄)

Here sending an empty list is clearly an error in the client's code, so the returned error is just for developers, and the errorcode in it does not matter at all. There is a general `DataValidationError` which is used in this kind of situations elsewhere in the code, so I would rather use that with some kind of an error message describing the error.

Reply:
Usually a problem is that there are too generic errors or no proper logging at all. Because there are pretty many specific error codes 1 does not hurt much more :) I too think that there are too specific codes, but in passwords handling there was followed this pattern already so...